ReactTurboModuleCxx/React-TurboModuleCxx-RNW.podspec

```diff
@@ -0,0 +1,39 @@
+require "json"
+
+package = JSON.parse(File.read(File.join(__dir__, "..", "package.json")))
+version = package['version']
+
+folly_compiler_flags = '-DFOLLY_NO_CONFIG -DFOLLY_MOBILE=1 -DFOLLY_USE_LIBCPP=1 -Wno-comma -Wno-shorten-64-to-32'
+folly_version = '2020.01.13.00'
+
+Pod::Spec.new do |s|
+  s.name                   = "React-TurboModuleCxx-RNW"
+  s.version                = version
+  s.summary                = "C++ supporting for Turbo Module."
+  s.homepage               = "https://github.com/microsoft/react-native-windows"
+  s.license                = package["license"]
+  s.author                 = "Microsoft Corporation"
+  s.platforms              = { :ios => "10.0", :tvos => "10.0", :osx => "10.13" }
+  s.compiler_flags         = folly_compiler_flags
+  s.source                 = { :git => 'https://github.com/microsoft/react-native-windows.git',
+                               :commit => "d9077991441889ddaa18a8af6a2cc8514ca7714d" }
+  s.source_files           = "vnext/Shared/TurboModuleRegistry.h",
+                             "vnext/Microsoft.ReactNative/JsiReader.{h,cpp}",
+                             "vnext/Microsoft.ReactNative/JsiWriter.{h,cpp}",
+                             "vnext/Microsoft.ReactNative/TurboModulesProvider.{h,cpp}",
+                             "vnext/Microsoft.ReactNative.Cxx/JSValue*.{h,cpp}",
+                             "vnext/Microsoft.ReactNative.Cxx/Module*.{h,cpp}",
+                             "vnext/Microsoft.ReactNative.Cxx/React*.{h,cpp}",
+                             "vnext/Microsoft.ReactNative.Cxx/StructInfo.h",
+                             "vnext/Microsoft.ReactNative.Cxx/NativeModules.h"
+  s.library                = "stdc++"
+  s.pod_target_xcconfig    = { "USE_HEADERMAP" => "YES",
+                               "CLANG_CXX_LANGUAGE_STANDARD" => "c++17" ,
+                               "HEADER_SEARCH_PATHS" => "\"$(PODS_ROOT)/RCT-Folly\"" "\"$(PODS_ROOT)/Headers/Private/React-callinvoker\"" "\"$(PODS_ROOT)/Headers/Private/ReactCommon\"" }
```

Comment:
Did you really need to have `$(PODS_ROOT)/RCT-Folly` even though you depend on `RCT-Folly` below?

As for the other two, looks like those should perhaps become public instead?

Reply:
fixed